cmd/aro/mirror.go

```diff
@@ -58,6 +59,11 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
+	srcauthRedhat, err := getAuth("SRC_AUTH_REDHAT")
```

Comment:
I see that we already use this pattern in other variable names, like `srcauthQuay` (not `srcAuthQuay`), but I don't see obvious reason for it, can you explain why we don't observe camel case here? This looks weird to me